frontend/components/synthesizePolygons/SynthesizePolygons.js

```diff
@@ -4,11 +4,10 @@ import PolygonViewer from "./PolygonViewer";
 import PolygonEditor from "./PolygonEditor";
```

Comment:
This needs a round of tidy-up: there's, e.g., state vars that are just constants now but are still state, lots of commented out code, etc., but I can go through and cleanup some of this later.